apps/accounts/models.py

```diff
@@ -546,3 +546,24 @@ def export_admin_log(sender, instance, **kwargs):
                 'action_time': instance.action_time}
 
         admin_logger.info(msg)
+
+
+def get_user_id_salt(salt=settings.USER_ID_SALT):
+    """
+    The settings.USER_ID_SALT value can be hexified (depends on environment).
+    We need to unhexify before using is it is hexified.
+    This function should be used to return the correct form of USER_ID_SALT
+    from the settings file based on USER_ID_HEX_STATUS
+    A value can be passed to this function, otherwise settings.USER_ID_SALT
+    is used.
+
+    :param: salt
+    :return: un_hex_salt
+    """
+
+    if settings.USER_ID_HEX_STATUS:
+        un_hex_salt = binascii.unhexlify(salt)
```

Comment:
Make this the only function in this, basically assume everything needs to be `unhexlify`ed

Reply:
Update `USER_ID_SALT` to be the hex string in all envs. This will have to happen in the ops repo.